### ISSUE TITLE
Camera2D near and  near and far types

### ISSUE THREAD

**Issue by alejcas** [pythonarcade/arcade#2057]
Hi, two related things:

1. It appears that calling Camera2D will default to near = 0.0, but calling Camera2D.from_raw_data will default to near = -100
2. near and far have float types but are set as integers by default by Camera2D.from_raw_data

I don't send a PR because I don't know if it should be 0.0 or -100.

**Comment by DigiDuncan** [comment]
Pinging @DragonMoffon, they're the ones handling Camera2D primarily.

**Comment by DragonMoffon** [comment]
For Camera2D it should be a float and -100.0 is correct.

**Comment by pushfoo** [comment]
The commit Dragon made wasn't cherry-pickable. I asked him in Discord whether it would be okay to roll into that  branch. He said yes, so:
1. I redid it on my branch (0f75155215d6714348798e92e1f3b97d9268dadd)
2. It's credited to him

**Comment by alejcas** [comment]
Closing this as it's fixed now

**Comment by pushfoo** [comment]
TL;DR: I disagree since it's not merged and there will be force pushes breaking commit hashes.

It's not merged or released yet, but you can use that commit hash if you'd like assuming you mirror the branch to your repo. Btw, I strongly suggest mirroring and pinning if you want to use that branch ASAP because there will be commit hash changes.

**Comment by alejcas** [comment]
> TL;DR: I disagree since it's not merged and there will be force pushes breaking commit hashes.
> 
> It's not merged or released yet, but you can use that commit hash if you'd like assuming you mirror the branch to your repo. Btw, I strongly suggest mirroring and pinning if you want to use that branch ASAP because there will be commit hash changes.

You are right, so reopening again

**Comment by pushfoo** [comment]
@alejcas Can you pull the latest `development` commits to confirm this is fixed?

**Comment by alejcas** [comment]
> @alejcas Can you pull the latest `development` commits to confirm this is fixed?

It is!
